packages/jsii/test/negatives/neg.internal-underscore.3.ts

```diff
@@ -0,0 +1,6 @@
+///!MATCH_ERROR: the name of members marked as @internal must begin with an underscore
+
+export interface IMyInterface {
+  /** @internal */
+  methodWithInternalButNoUnderscore: string;
```

Comment:
There is no difference between this test and neg 1.

Reply:
Was supposed to be a class. Will fix. Thanks

Reply:
Fixed.